controllers/admin/AdminImagesController.php

```diff
@@ -98,7 +98,7 @@ public function init()
 
         $this->canGenerateAvif = $this->get('PrestaShop\PrestaShop\Core\Image\AvifExtensionChecker')->isAvailable();
         $this->isMultipleImageFormatFeatureEnabled = $this->get(FeatureFlagRepository::class)->isEnabled(FeatureFlagSettings::FEATURE_FLAG_MULTIPLE_IMAGE_FORMAT);
-        $this->imageFormatConfiguration = $this->get(ImageFormatConfiguration::class);
+        $this->imageFormatConfiguration = $this->get('PrestaShop\PrestaShop\Core\Image\ImageFormatConfiguration');
```

Comment:
Why not keeping this one?
```suggestion
        $this->imageFormatConfiguration = $this->get(ImageFormatConfiguration::class);
```